lib/reporting/protocols_report.rb

```diff
@@ -246,13 +246,19 @@ def saml_signature_issues_table
       ]
     end
 
-    def loa_acr_requests_table
+    def deprecated_parameters_table
       [
-        ['Count of issuers using LOA', 'List of issuers with the issue'],
+        ['Deprecated Parameter', 'Count of issuers using the parameter', 'List of issuers using the parameter'],
         [
+          'LOA',
           loa_issuers_data.length,
           loa_issuers_data.join(', '),
         ],
+        [
+          'AAL3',
+          aal3_issuers_data.length,
+          aal3_issuers_data.join(', '),
```

Comment:
is this report usable with all these issues in the same cell like that? they're very long values

IMO if I were making a report like this, I'd have separate tables per attribute, and the table would be like

| Friendly Name | Issuer |
| --- | --- |
| Foobar.gov | foo:bar:baz|


etc

Reply:
I had suggested something similar when the original story was discussed, but it was turned down.